hail/python/hail/methods/qc.py

```diff
@@ -285,9 +285,51 @@ def has_field_of_type(name, dtype):
     return mt.annotate_rows(**{name: result})
 
 
+def concordance2(left, right, *, _localize_global_statistics=True):
+    print('conc2')
```

Comment:
The print is clearly debugging code. Is `concordance2` itself also?

Reply:
whoops! Yes, the whole function can be removed.